src/views/Project/index.jsx

```diff
@@ -271,14 +271,14 @@ export default class Project extends Component {
     });
   };
 
-  load = async () => {
+  load = () => {
```

Comment:
Hi, can you elaborate the reason you remove the `async` from the function?

Reply:
Yes, when it was called by loadNextPage, Promise.all for fetchGithub had issues with it being async and awaited, causing it to load the same query from the same cursor twice, as opposed to loading a query and loading the one after with the new cursor. After 2 tries, loadNextPage just gave up. Due to this github data only displayed the first query. I got rid of the await on the Promise.all and that fixed it. I got rid of await on everything else in that function too after, even though fetchBugzilla wasn't having the same problems at the moment, I thought it might in the future. When I got rid of all the awaits, the async was no longer needed, and I believe the function works perfectly now.